homeassistant/components/update/__init__.py

```diff
@@ -104,6 +104,11 @@ async def handle_update(
             backup=msg.get("backup"),
         )
     except Exception as err:  # pylint: disable=broad-except
+        _LOGGER.exception(
```

Comment:
Since we log the stack trace here, it would be great if we could introduce an `UpdateFailed` exception that won't print a stack trace but still results in an update_failed error.

Reply:
Make sense :+1: